.github/workflows/validate_packs.yml

```diff
@@ -21,3 +21,25 @@ jobs:
       - name: Validate files
         id: validate-files
         run: yarn validate-packs 
+
+  ensure-pack-names-are-unique:
```

Comment:
We might want to update the previous step to make it clear that it's only _part_ of the validation. Perhaps something about schema compliance.